packages/stream_chat/lib/src/api/channel.dart

```diff
@@ -296,40 +297,30 @@ class Channel {
   /// Send a [message] to this channel.
   /// Waits for a [_messageAttachmentsUploadCompleter] to complete
   /// before actually sending the message.
-  Future<SendMessageResponse> sendMessage(Message message) async {
+  Future<SendMessageResponse?> sendMessage(Message message) async {
```

Comment:
I think the return type for all the network call functions can be marked non-nullable.